File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    let answer = [];
    let newDict = {};
    elements = s.slice(2, s.length-2).split('},{')
    elements = elements.map(function(element) {
        return element.split(',').map(function(num) {
            return parseInt(num);
        });
    });
    elements.sort((a,b) => a.length - b.length)
    for (let idx = 0; idx < elements.length; idx++) {
        for (let i = 0; i < elements[idx].length; i++) {
            if (elements[idx][i] in newDict) {
                newDict[elements[idx][i]] += 1
            } else{
                newDict[elements[idx][i]] = 1
            }
        }
    }
    let sortedKeys = Object.keys(newDict).sort((a, b) => newDict[b] - newDict[a]);

    for (let key of sortedKeys) {
        answer.push(parseInt(key))
    }

    return answer;
}

File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s):
    answer = []
    # 파싱
    elements = s[2:-2].split('},{')
    elements = [list(map(int, element.split(','))) for element in elements]
    # 파싱 완료
    elements.sort(key=len)
    newDict = dict()
    for idx in range(len(elements)):
        for id in elements[idx]:
            if id in newDict:
                newDict[id] += 1
            else:
                newDict[id] = 1

    for idx in newDict:
        answer.append(idx)
    return answer